data-prepper-plugins/http-common/src/main/java/org/opensearch/dataprepper/plugins/server/CreateServer.java

```diff
@@ -40,17 +39,17 @@
 import java.io.ByteArrayInputStream;
 import java.nio.charset.StandardCharsets;
 import java.time.Duration;
+import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
 import java.util.concurrent.ScheduledThreadPoolExecutor;
 import java.util.function.Function;
 
-
 public class CreateServer {
```

Comment:
I am not so sure about the class name. To me this class looks like a factory of some kind and I would name it as such. But I am also fine with the name it got right now.

Reply:
Hi Karsten, 
The CreateServer abstaction was added in last week by this PR https://github.com/opensearch-project/data-prepper/pull/5653. I can rename it `ServerFactory` or something similar, but I feel this can be out of scope for this PR.

Reply:
Yeah, I have seen when the change was introduced. I would appreciate a renaming at some point in time, but I am fine with not doing it in this PR.